api/config.py

```diff
@@ -185,9 +185,9 @@ class Configuration(CoreConfiguration):
             "key": PATRON_WEB_CLIENT_URL,
             "label": _("URL of the web catalog for patrons"),
```

Comment:
This is no longer a good enough description for this setting, if it ever was. Both the `label` and the `description` need to be updated.

What is this now? I think it's something like:

Hostnames for web application access

Only web applications from these hosts can access this circulation manager.

This can be a single hostname ("catalog.library.org"), or a regular expression that matches many hostnames (".*\.libraryvendor\.com"). You can also set this to "*" to allow access from any host, but you must not do this in a production environment -- only during development.

Reply:
Cool, just updated the label and description to what you suggested